tests/unit/monitor/api/report/test_report_parallel.py

```
@@ -0,0 +1,121 @@
+
+            api._get_report_data_parallel(threads=4)
+
+            mock_pool_cls.assert_called_with(max_workers=4)
+
+    def test_error_propagation(self, mock_dbt_runner):
+        api = ReportAPI(mock_dbt_runner)
+        error = RuntimeError("test error")
+        with patch.object(
+            api, "_create_subprocess_runner", side_effect=error
+        ):
+            result, err = api._get_report_data_parallel(threads=4)
```

Ruff (0.15.10)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)